src/js/components/AppPageComponent.jsx

```diff
@@ -182,6 +182,26 @@ var AppPageComponent = React.createClass({
     }
   },
 
+  getUnhealthyTaskMessage: function (healthCheckResults = []) {
+    return healthCheckResults.map((hc, index) => {
+      if (hc && !hc.alive) {
```

Comment:
Can we use descriptive variable names? `hc` just requires the reader to search the code around or the calling logic to figure out what an abbreviation means.  Calling it `healthCheck` is easy enough.